run-reports.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+export TASKCLUSTER_ROOT_URL=https://firefox-ci-tc.services.mozilla.com/
+export REPORT_SCHEDULER_ID="smoketest"
+export REPORT_PREFIX=https://github.com/taskcluster/mozilla-history/blob/master/WorkerVersions/
+
+# needs to be set in the environment:
+# TASKCLUSTER_CLIENT_ID
+# TASKCLUSTER_ACCESS_TOKEN
+
+set -euo pipefail
+
+git pull
+
+echo "generating audit report"
+./mozilla-history
```

Comment:
`./mozilla-history` suggests mozilla-history is in the same directory, but that depends on how it was built. Maybe it should build it first (which exits in a few milliseconds if it is already built) - i.e. with a `go build -o mozilla-history` or so first? Otherwise no guarantee that `mozilla-history` binary is in this directory.

Reply:
the problem here if you run it from the community-history repo, it will not have this code, so it is kind of a dependency that needs to be built first

Maybe we can do the `go install github.com/taskcluster/...` here to make sure it is present?

Reply:
The issue is that you have `./mozilla-history` which implies it is in the same repo. If it isn't in the same repo but just needs to be in the `PATH` then this should really be `mozilla-history` instead of `./mozilla-history`.

Reply:
Yeah, I think they are quite similar but we end up copy-pasting things back and forth.. maybe it's even worth merging both repos and have per-deployment folder structure. 

Or keep scripts in single place and explain in documentation how to access and set it up

Reply:
agree single repo is probably better, and a subdirectory for each environment. nice idea!